_layouts/donate.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: base
+---
+<div class="container">
+    <div class="bg" style="background-image: url({{ page.background_image }});">
+        <div class="spacing"></div>
+        <div class="row">
+            <div class="col-sm-1"></div>
+            <div class="col-sm-7">
+                <script src="https://donorbox.org/widget.js" paypalExpress="false"></script><iframe allowpaymentrequest="" frameborder="0" height="900px" name="donorbox" scrolling="no" seamless="seamless" src="https://donorbox.org/embed/bab-surya-test-campaign" style="max-width: 500px; min-width: 310px; max-height:none!important" width="100%"></iframe>
```

Comment:
Can you format this line so that the line is not too long to read?

Reply:
Can we also make the value of `src="https://donorbox.org/embed/bab-surya-test-campaign"` to be taken from the frontmatter file (`.md` file)

Reply:
Done.